src/services/DocumentVerifier.ts

```diff
@@ -1,6 +1,8 @@
 import { verifyWithIndividualChecks } from "@govtechsg/oa-verify";
-import { SignedDocument } from "@govtechsg/open-attestation";
+import { verify, isValid } from "@govtechsg/opencerts-verify";
+import { SignedDocument, WrappedDocument } from "@govtechsg/open-attestation";
```

Comment:
Remove the use of SignedDocument it's not valid ...

I also have typing issue here => WrappedDocument does not exist => need to update open-attestation lib